# Proyecto Automovil

Hemos utilizado los metodos try catch y throw para realizar el control de errores de la aplicación.

La declaración **try...catch**  las instrucciones a intentar en el **try**, y especifica una respuesta si se produce una excepción **catch**.

Un bloque **catch** contiene sentencias que especifican que hacer si una excepción es lanzada en el bloque **try**. Si cualquier sentencia dentro del bloque try (o en una funcion llamada desde dentro del bloque try) lanza una excepción, el control cambia inmediatamente al bloque catch . Si no se lanza ninguna excepcion en el bloque try, el bloque catch se omite.